datagenerator/datagenerator.iml

```diff
@@ -0,0 +1,365 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be checked in and should be in `.gitignore`. IntelliJ files can be generated from `gradle idea` command.